spec/support/activity_form.rb

```diff
@@ -503,4 +510,11 @@ def fill_in_tags
     end
     click_button I18n.t("form.button.activity.submit")
   end
+
+  def fill_in_commitment
+    optional = activity.third_party_project? ? "" : " (optional)"
+    expect(page).to have_content I18n.t("page_title.activity_form.show.commitment", optional: optional)
```

Comment:
The `optional` variable doesn't appear to be in the translation, so I think this argument is being ignored and it's always returning "Original commitment figure"

Beyond fixing the translation, is it worth testing that " (optional)" doesn't appear at levels B and C to catch this?

Should we also use a translation for "(optional)", maybe somewhere more generic than the activity translations?

Reply:
I think this is actually working here, but as you pointed out below, I added it in the wrong commit. Have fixed this.

I've used a translation for optional in the code itself, but left it here as part of the new trend towards using real text in tests.